CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# CONTRIBUTING
+
+Bug reports and pull requests are welcome on GitHub at https://github.com/Shopify/ruby-lsp-rails. This project is
+intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the
+[Contributor Covenant](https://github.com/Shopify/ruby-lsp-rails/blob/main/CODE_OF_CONDUCT.md) code of conduct.
+
+# Developing on Ruby LSP
+
+For general information about developing, refer to the to the [documentation](https://github.com/Shopify/ruby-lsp/blob/main/CONTRIBUTING.md#debugging-with-vs-code) for Ruby LSP itself.
+
+Ruby LSP uses [Sorbet](https://sorbet.org/) for typechecking, but if you're not familiar with it then it's ok to submit
+a PR without any type annotations, and the maintainers can add them.
```

Comment:
Sorry, I'm not sure I agree with the wording here. Maybe we can say, something along the lines of "if you're not familiar with Sorbet, our team can help you".

We won't accept PRs that don't pass type checking and we're not going to be able to push code ourselves to contributor's forks.